react-native/react/tabs/devices/index.js

```diff
@@ -47,12 +50,27 @@ export default class Devices extends Component {
   }
 
   render () {
-    return <DevicesRender devices={this.props.devices} showRemoveDevicePage={this.props.showRemoveDevicePage} showExistingDevicePage={this.props.showExistingDevicePage} showGenPaperKeyPage={this.props.showGenPaperKeyPage} />
+    const {
+      devices,
+      waitingForServer,
+      showRemoveDevicePage,
+      showExistingDevicePage,
+      showGenPaperKeyPage
+    } = this.props
+
+    return <DevicesRender {... {
+      devices,
+      waitingForServer,
+      showRemoveDevicePage,
+      showExistingDevicePage,
+      showGenPaperKeyPage
+    }}/>
```

Comment:
especially when its mixed with derived props and etc. exploding / imploding it i dunno, double the lines...